app/models/government.rb

```diff
@@ -20,4 +22,51 @@ def self.on_date(date)
   def current?
     self == Government.current
   end
+
+  def overlaps?(other)
+    !(is_before?(other) || is_after?(other))
```

Comment:
I'm wondering if this would be more easily understood as `!is_before?(other) && !is_after?(other)`. In my head, that would read as "it's not before and it's not after", which I think is easier to comprehend.